plugins-bundled/internal/marketplace-app/src/components/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2, stylesFactory } from '@grafana/ui';
+
+interface Props {
+  href: string;
+  text: React.ReactNode;
+  image: React.ReactNode;
+  layout?: 'horizontal' | 'vertical';
+}
+
+export const Card = ({ href, text, image, layout = 'vertical' }: Props) => {
+  const theme = useTheme2();
+  const styles = getCardStyles(theme, layout);
```

Comment:
Maybe use `useStyles2` instead?

Reply:
☝️

Reply:
I don't think I can do that here because I'm passing the layout prop to decide how flexbox should layout the contents of the card. 🤔

Reply:
Ok, gotcha!